core/src/test/java/org/infinispan/xsite/AbstractXSiteTest.java

```diff
@@ -66,12 +72,16 @@ protected void destroy() {
 
    private void killSites() {
       for (TestSite ts : sites) {
-         TestingUtil.killCacheManagers(ts.cacheManagers);
+         killSite(ts);
       }
       sites.clear();
       siteName2index.clear();
    }
 
+   protected void killSite(TestSite ts) {
+      TestingUtil.killCacheManagers(ts.cacheManagers);
```

Comment:
you can't use the ```killCacheManagers``` method. It will stop the ```__acl_cache``` too soon and fail to destroy the site.